src/base/detail/InitializedValue.hh

```diff
@@ -0,0 +1,57 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file InitializedValue.hh
+//---------------------------------------------------------------------------//
+#pragma once
```

Comment:
Just a question, should we modify these in Shift as well?  I know this is technically non-standard, but widely (universally?) supported.

Reply:
Honestly now that I'm used to seeing them in code, I like it better -- it trims the file length by a few lines, you don't have to worry about updating the template names. It seems like they're universally supported and even account for symlinks.

I think the SO page here has a good discussion: https://stackoverflow.com/questions/787533/is-pragma-once-a-safe-include-guard and it seems like there's a consensus on `#pragma once` basically being preferred.

Reply:
I definitely like it better actually!